### PR TITLE
Redirect to login page on 401

### PR DESCRIPTION
**Pull request by andyjmaclean**

### CODE REVIEW

File: projects/metis/src/app/_services/error.interceptor.ts [hunk 1]
```diff
@@ -27,9 +27,20 @@ export class ErrorInterceptor implements HttpInterceptor {
...
-      .pipe(retry({ count: this.numberOfRetries, delay: this.shouldRetry }));
+    return handler.handle(request).pipe(
+      retry({ count: this.numberOfRetries, delay: this.shouldRetry }),
+      tap({
+        error: (res) => {
+          if (res.status === 401) {
+            this.expiredToken();
+          }
+          if (res.status === 406) {
+            console.log('got the 406');
```
Comment (jeortizquan) [inline projects/metis/src/app/_services/error.interceptor.ts:38]: clean up this?

Comment (jeortizquan) [reply projects/metis/src/app/_services/error.interceptor.ts:38]: or `expired token` message

**Review by jeortizquan — COMMENTED** [review]
Just a small comment if needs clean up or not.

File: projects/metis/src/app/_services/error.interceptor.ts (hunk 1 above)
Comment (andyjmaclean) [reply projects/metis/src/app/_services/error.interceptor.ts]: Done!

**Review by jeortizquan — APPROVED** [review]

**Pull request merged**